packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -130,7 +130,7 @@ export abstract class SharedObjectCore<TEvent extends ISharedObjectEvents = ISha
      * to variables to avoid complaints from TypeScript.
      */
     private setUpSampledTelemetryHelpers(): SampledTelemetryHelper[] {
-        assert(this.mc !== undefined && this.logger !== undefined, "this.mc and/or this.logger has not been set");
```

Comment:
Was this one part of the cherry-pick to get the pipelines to work? I wouldn't expect the assert tagging to have happened as part of those changes, nor as part of this PR. If it did come from the cherry pick, might be fine but need to think more about it.